CruCentralCoast/Community/Missions/MissionDetailsVC.swift

```diff
@@ -30,7 +33,17 @@ class MissionDetailsVC: UIViewController {
     
     @IBAction func learnMore() {
         // TODO
-        self.presentAlert(title: "Learn More", message: "Coming Soon...")
+        
+        if let url = URL(string: missionURLString!) {
```

Comment:
Definitely don't force unwrap this. Not all missions will contain URLs. The app crashes when tapping the Learn More button on the last Mission

Reply:
I totally forgot to fix this but i completely agree! Thanks for the catch